templates/tableau-single-server-ubuntu.template

```diff
@@ -602,22 +565,20 @@
                 }
             }
         },
-        "IPAssoc": {
-            "Type": "AWS::EC2::EIPAssociation",
-            "Condition": "HasElasticIP",
+        "EIP": {
+            "Type": "AWS::EC2::EIP",
```

Comment:
Random question from the public. I noticed this change forces the template to create a new Elastic IP during the Workload stack creation. Is this correct?

I'm in a situation where I need to associate to an existing elasticIP because I have hit the limit of Elastic IPs. Looks like I can use this code removed here instead.

Reply:
(and noted this PR is a few years old)

Reply:
@lucaswiley, I happened to note this comment, so will reply. In future it might be better to create a new issue with your question for visibility. You are correct, for the single node configuration we've added an EIP since the deployment is in the public subnet. For the cluster configuration we use a load balancer and EIP is not used. You can fork the code and make any edits that suit better for your use case. You may also be able to ask for a limit increase by raising a support request.

Reply:
Awesome, thank you for clarifying. Yeah, we ended up just increasing our limit to resolve. Thank you and enjoy your weekend.